### DISCUSSION TITLE
Issue with Automatic Loading of DICOM Files in Weasis 4.2.0

### DISCUSSION THREAD

**Discussion by simas112** [nroduit/Weasis#471] [Q&A]
Hello, 

I am currently using Weasis DICOM Viewer version 4.2.0 and am encountering an issue with the automatic loading of DICOM files upon startup. I have integrated Weasis into my software and am trying to export studies into the archive with the viewer. 

Based on what I have gathered from the Weasis\app\conf\base.json file, it appears that the weasis.portbale.dicom.directory value is set to "DICOM". My understanding from this is that Weasis should read and automatically load the DICOM files present in the DICOM folder upon startup. However, I am finding that the studies are not loaded after the startup process. 

Here's what I have tried and verified: The DICOM files are placed in the designated "DICOM" folder. The base.json file configuration pointing towards the correct directory. Attempted to manually load the DICOM files into Weasis with success, confirming that the files themselves do not seem to be the issue. Despite the above, the automatic loading of the studies is not occurring as expected. Is there a step or configuration that I might be missing, or is there an additional setup required to enable this functionality? Any insights or advice would be greatly appreciated.

**Comment by nroduit** [comment]
This feature is only used when the viewer is embedded on CDs.

To load images at startup, simply use the commands, see https://weasis.org/en/getting-started/weasis-protocol/index.html#examples-to-load-images

**Comment by simas112** [comment]
Thank you for getting back to me so quickly! I understand that the feature is utilized when the viewer is embedded on CDs. However, our use case involves distributing the viewer via USB drives, where we upload .zip archives containing the DICOM studies and the viewer. Is it possible to enable or adapt the auto-loading feature for studies contained within .zip archives on USB drives, or is there an alternative approach that we could take to achieve a similar user experience?

**Comment by nroduit** [comment]
DICOM CDs and DICOM ZIPs are 2 different things defined by the DICOM standard. So if you want to make a DICOM CD on USB media, you need to have a DICOMDIR at the root and you cannot use DICOM ZIP.

If the idea of the zip is to compress the files, it doesn't seem to me to be the right approach because lossless compression on the image will always be better. You can see the [DICOM CD/DVD build with Weasis](https://weasis.org/en/tutorials/dicom-export/index.html#cddvd-image) where you can apply compression in the options. 

If the aim is to open a DICOM ZIP with Weasis, then follow the instructions with the commands.
